src/main/java/net/sf/jabref/event/ChangeEntryEvent.java

```diff
@@ -0,0 +1,9 @@
+package net.sf.jabref.event;
+
+public class ChangeEntryEvent extends AddOrChangeEntryEvent {
+
+    public ChangeEntryEvent(Object object) {
```

Comment:
An Entry is always a BibEntry. Please rely on Java's type checking and do not use generic types and cast internally.